vignettes/duckplyr.Rmd

```diff
@@ -0,0 +1,89 @@
+---
+title: "duckplyr"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{duckplyr}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+```{r setup}
+library(duckplyr)
+```
+
+## Design principles
+
+The duckplyr package uses **DuckDB under the hood** but is also a **drop-in replacement for dplyr**.
```

Comment:
this comes from the blog post draft :sweat_smile:

Reply:
Not the future post on duckplyr, the post on laziness https://github.com/r-hub/blog/pull/179